scripts/lib/libvirt/fixtures/cloud-node1-uefi.xml

```diff
@@ -5,8 +5,8 @@
   <vcpu>1</vcpu>
   <os>
     <type arch='x86_64' machine='pc-0.14'>hvm</type>
-<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-ms-code.bin</loader>
-    <nvram>/var/lib/libvirt/cloud-node1_VARS.fd</nvram>
+<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-code.bin</loader>
```

Comment:
why are we switching to the non-secureboot loader? I think it would make more sense to test the secureboot case